.github/ISSUE_TEMPLATE/server-migration.md

```diff
@@ -9,6 +9,8 @@ assignees: ''
 
 Checklist based on general guide https://github.com/openfoodfoundation/ofn-install/wiki/Migrating-a-Production-Server
 
+Tip: find/replace to set up most commands ready to go, eg: `x_prod` -> `ca_prod`
```

Comment:
I wonder if we should use easy to replace example names like `old_server` and `new_server` which could be prod or staging. Some of the guide just refers to `x` instead of `x_prod` as well. It would be good to make this consistent.